app/__init__.py

```diff
@@ -1,7 +1,13 @@
 from flask import Flask
 from .db import db, migrate
 from .models import task, goal
+from .routes.task_routes import tasks_bp
 import os
+from dotenv import load_dotenv
+from .routes.goal_routes import goals_bp
```

Comment:
Feel free to get in the habit of naming all our blueprints `bp` within their routes and aliasing them within the init.py